equipment/models.py

```diff
@@ -48,17 +43,40 @@ class EquipmentModel(models.Model):
 
     def __str__(self):
         return self.name
+		
+class VirusBatch(models.Model):
+# i took out "provided by a supplier" because we make these ourselves (they are diluated from what the supplier supplies)    
+# might also need a location field (e.g. which fridge it is in) - let's ask Charu
+	"""
+    A virus batch
+    """
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    virus_type =  models.CharField(max_length=255, null=True, blank=True,
+                                   help_text="UPenn ID or equivalent")
+    description = models.CharField(max_length=255, null=True, blank=True)
+    virus_source = models.ForeignKey('VirusSource', null=True, blank=True
+                                    help_text="Who supplied the virus")
+    date_time_made = models.DateTimeField(null=True, blank=True, default=datetime.now)
+    nominal_titer = models.FloatField(null=True, blank=True, help_text="TODO: What unit?") 
+	# let's ask Charu about what unit.
+
+    class Meta:
+        verbose_name_plural = "virus batches"
 
 ###############################################################################
 ### Appliances
 ###############################################################################
 
 class Appliance(PolymorphicModel):
+	# If someone buys a new piece of equipment that doesn't belong in any of the subclasses, can they just add it 
+	# as an Appliance, with no subclass?
+	#
+	# also, when you have two of one appliance, do they both get entries here?
```

Comment:
At the moment, you can't add an Appliance directly, only one of the subclasses. I don't know whether to rethink the way we do this... possibly making this an abstract BaseAppliance class and having a GenericAppliance subclass for things which don't fit into the category? But each field should reference a specific type of appliance.

Also yes, each appliance has a record here.